Whos-Who.md

```diff
@@ -1 +1,56 @@
 # Who's who in the terra team
+
+##Current team members in no particular order 
+
+<table>
+    <tr>
+        <th>Name</th>
+        <th>Role</th>
+        <th>Pronouns</th>
+    </tr>
+    <tr>
+        <td>Amelia Sagoff</td>
+        <td>Product Owner</td>
```

Comment:
TOL: We may want to add a roles description doc and link some of the specialized roles to that. (Product Owner, Delivery Lead, Tech Principle, etc.)

Reply:
I like the idea of having more descriptions of the roles, I think that would be a great addition. To my knowledge that doesn't yet exist as a document, so that might be something that needs to be written up. 

I think the .idea files were automatically generated for some reason by intelliJ (maybe it always does that?). I agree that they don't need to be in here and could be added to gitignore.